sw/device/tests/sensor_ctrl_alerts.c

```diff
@@ -182,6 +169,26 @@ bool test_main(void) {
                                              /*ecc_en*/ false,
                                              /*he_en*/ false);
 
+  // If all bits are set, then this must be the first iteration
```

Comment:
Extend  the comment to indicate on the first iteration you set events_vector to a random number, with the bits set becoming the alerts to be tested, and then you progressively clear them when incrementing.